Add files via upload

Turtlefud12 opened the pull request:
here is the arrival process class, the customer class, the custom queue class, the job class, and the node class

Turtlefud12 committed:
Add files via upload

here is the arrival process class, the customer class, the custom queue class, the job class, and the node class

hollymetzger reviewed (changes requested):
Job doesn't need a setServiceTime() because that value shouldn't be changed in the middle of the simulation

Turtlefud12 committed:
Update Job.java

took out the get service time bits, recheck over l8r <3 Beth

Turtlefud12 committed:
Update Job.java

oopssies I took out the get service time instead of the set service time, so i fixed it and took out the set service time

Turtlefud12 commented:
I fixed it <3
